2024/week0_course_primer.md

```diff
@@ -72,6 +72,10 @@ Throughout this course, we will provide you with a set of reflection questions t
 3. What distinguishes interactive from non-interactive proofs?
 :::
 
+## Practical
+
+Many zk proof systems are written in Rust. Learn Rust with [Rustlings](https://rustlings.cool/). It's a game with nearly 100 challenges. You are required to finish it.
```

Comment:
```suggestion
Since many ZK proof systems are written in Rust, take the time to learn Rust with [Rustlings](https://rustlings.cool/). It is a game with nearly 100 challenges. You are required to finish it before the start of the program.
```

Reply:
Another suggestion here.